client/web/src/integration/blob-viewer.test.ts

```diff
@@ -598,7 +598,7 @@ describe('Blob viewer', () => {
             )
         })
 
-        it('sends the latest document to extensions', async () => {
+        it.only('sends the latest document to extensions', async () => {
```

Comment:
```suggestion
        it('sends the latest document to extensions', async () => {
```

Unless this is what fixes the flake 😉